substrate/client/transaction-pool/src/common/tracing_log_xt.rs

```diff
@@ -0,0 +1,48 @@
+macro_rules! log_xt {
```

Comment:
Please add license header + little doc.
```suggestion
 // This file is part of Substrate.

 // Copyright (C) Parity Technologies (UK) Ltd.
 // SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
 // This program is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
 // the Free Software Foundation, either version 3 of the License, or
 // (at your option) any later version.

 // This program is distributed in the hope that it will be useful,
 // but WITHOUT ANY WARRANTY; without even the implied warranty of
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
 // GNU General Public License for more details.

 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.

 //! Utility for logging transaction collections with tracing crate.
macro_rules! log_xt {
```